source/manual/zendesk.html.md

```diff
@@ -23,7 +23,10 @@ relevant and actionable - and start with the oldest by date. If external assista
 line is required, assign them on the ticket and move on to the next ticket (keeping the ticket in the
 2nd line queue).
 
-Ignore 'Low' priority tickets - these have been triaged by 2nd line for GOV.UK product managers to review.
+'Low' priority tickets are for tickets that are:
+• `on-hold` or are waiting to be handed over to another team (unless they are product related, they
+  can be assigned to `3rd line product`)
+• tickets that have been dealt with and can be closed if the user doesn't reply in 5 working days
```

Comment:
I don't think the bullet points are rendering correctly.

Reply:
Thanks @leenagupte , I think that's fixed now!